Paco-iOS/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
We don't currently use CocoaPods. We played around with it but haven't made the change from our local build using vendor/* lib dependency management. This can stay as is, but, just an FYI